### ISSUE TITLE
bug: Can't log with oidc via Keycloak

### ISSUE THREAD

**Issue by ghost** [uxbox/uxbox#2875]
### Steps To Reproduce

1. Setup penpot as descibed in the docs
2. Create Keycloak client
3. Configure Penpot for oidc
```
# Frontend
- PENPOT_FLAGS=enable-registration enable-login-with-password enable-login-with-oidc
# Backend
- PENPOT_FLAGS=enable-registration enable-login disable-email-verification enable-smtp enable-prepl-server enable-login-with-oidc
- PENPOT_OIDC_BASE_URI=https://example.com/realms/example/
- PENPOT_OIDC_CLIENT_ID=penpot
- PENPOT_OIDC_CLIENT_SECRET=xxxxxxxxxxxxxxxxxxxxxxxx
- PENPOT_OIDC_AUTH_URI=https://example.com/realms/example/protocol/openid-connect/auth
- PENPOT_OIDC_TOKEN_URI=https://example.com/realms/example/protocol/openid-connect/token
- PENPOT_OIDC_USER_URI=https://example.com/realms/example/protocol/openid-connect/userinfo
```
5. Run penpot
`docker compose up -d`
6. Login via oidc

### Expected behavior

No error message? Rather a clean login?

### Actual behavior

The redirect to Keycloak works fine, but after login I get an error in the Penpot backend. The error is the reason, why the login fails.

### Environment (please complete the following information)

Self-hosted

OS: `Ubuntu 22.04 TLS`
Docker: `Docker version 20.10.23, build 7155243`
Compose: `Docker Compose version v2.15.1`

Applications
Penpot: `1.17.0`
Keycloak: `20.0.3`
Traefik: `2.9.6`

### Frontend Stack Trace

```console
172.10.10.6 - - [31/Jan/2023:22:14:10 +0100] "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
172.10.10.6 - - [31/Jan/2023:22:14:11 +0100] "GET /api/rpc/query/profile HTTP/1.1" 200 79 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
172.10.10.6 - - [31/Jan/2023:22:14:11 +0100] "GET /api/rpc/query/profile HTTP/1.1" 200 79 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
172.10.10.6 - - [31/Jan/2023:22:14:12 +0100] "POST /api/auth/oauth/oidc HTTP/1.1" 200 417 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
172.10.10.6 - - [31/Jan/2023:22:14:12 +0100] "GET /api/auth/oauth/oidc/callback?state=eyJhbGciOiJBMjU2S1ciLCJlbmMiOiJBMjU2R0NNIn0.GMpznuXP3DnNWuhaE4FMGJMUKXNQ-NDyesbi0XYKon7pP65JFN-5PQ.OKMnLnzAvOtJuqYw.ha4VS0pEnbEqYku_9KNtkrEHb5Ybu-ZX-bUPus9aBtEu3yStkljVJdp05lbbfdRM9jdqQyGREq_VMSLn6-MHnFJibcOn2a4k5N7DLZz8gegv5mPJbN4Y3_wvdZyc.O0hzj-ZIv2i3MJzK7VExgQ&session_state=0499f94f-ebe4-4519-8703-b0e661588756&code=ccebc68c-d421-4144-87c0-65996e982e85.0499f94f-ebe4-4519-8703-b0e661588756.1c4fc060-71dc-477c-9aac-dd882747d367 HTTP/1.1" 302 0 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
172.10.10.6 - - [31/Jan/2023:22:14:13 +0100] "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
172.10.10.6 - - [31/Jan/2023:22:14:13 +0100] "GET /api/rpc/query/profile HTTP/1.1" 200 79 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0"
```

### Backend Stack Trace

```console
+ exec /opt/jre/bin/java -jar penpot.jar -m app.main
[2023-01-31 21:13:36.339] I app.auth.oidc - hint="provider initialized", provider=:oidc, method="manual", client-id="penpot", client-secret="*********************", scopes="profile,email,openid", auth-uri="https://example.com/realms/example/protocol/openid-connect/auth", user-uri="https://example.com/realms/example/protocol/openid-connect/userinfo", token-uri="https://example.com/realms/example/protocol/openid-connect/token", roles-attr=nil, roles=nil
[2023-01-31 21:13:36.342] I app.metrics - action="initialize metrics"
[2023-01-31 21:13:36.381] I app.db - hint="initialize connection pool", name="main", uri="postgresql://penpot-postgres/penpot", read-only=false, with-credentials=true, min-size=0, max-size=60
[2023-01-31 21:13:36.684] I app.redis - hint="initialize redis resources", uri="redis://penpot-redis/0", io-threads=4, worker-threads=4, connect?=nil
[2023-01-31 21:13:36.834] I app.msgbus - hint="initialize msgbus", buffer-size=128
[2023-01-31 21:13:37.305] I app.setup.builtin-templates - hint="loading template files", total=9
[2023-01-31 21:13:37.713] I app.http - hint="starting http server", port=6060, host="0.0.0.0", name="http"
[2023-01-31 21:13:37.730] I io.undertow - starting server: Undertow - 2.3.0.Final
[2023-01-31 21:13:37.736] I org.xnio - XNIO version 3.8.8.Final
[2023-01-31 21:13:37.742] I org.xnio.nio - XNIO NIO Implementation Version 3.8.8.Final
[2023-01-31 21:13:37.761] I org.jboss.threads - JBoss Threads version 3.5.0.Final
[2023-01-31 21:13:37.826] I app.loggers.zmq - hint="receiver started"
[2023-01-31 21:13:37.831] I app.loggers.database - hint="initializing database error persistence"
[2023-01-31 21:13:37.838] I app.storage.tmp - hint="started tmp file cleaner"
[2023-01-31 21:13:37.860] I app.worker - hint="registry initialized", tasks=13
[2023-01-31 21:13:37.865] I app.worker - hint="cron: started", tasks=8
[2023-01-31 21:13:37.895] I app.worker - hint="dispatcher: started"
[2023-01-31 21:13:37.898] I app.worker - hint="monitor: started", name="default"
[2023-01-31 21:13:37.901] I app.worker - hint="worker: started", worker-id=0, queue="default"
[2023-01-31 21:13:37.902] I app.worker - hint="worker: started", worker-id=0, queue="webhooks"
[2023-01-31 21:13:37.903] I app.srepl - msg="initializing repl server", name="prepl", port=6063, host="localhost"
[2023-01-31 21:13:37.911] I app.main - hint="welcome to penpot", flags="login-with-password,backend-api-doc,secure-session-cookies,login,backend-worker,login-with-oidc,smtp,registration,prepl-server", worker?=true, version="1.17.0-9290-gd31138db7"
[2023-01-31 21:14:12.997] E app.auth.oidc - hint="error on oauth process"
TRACE:
 →  javax.net.ssl.SSLProtocolException: Cannot decode named group: x25519 (:-1)
DETAIL:
 →  javax.net.ssl.SSLProtocolException: Cannot decode named group: x25519 (:-1)
    at: sun.security.ssl.Alert.createSSLException(:-1)
        sun.security.ssl.Alert.createSSLException(:-1)
        sun.security.ssl.TransportContext.fatal(:-1)
        sun.security.ssl.TransportContext.fatal(:-1)
        sun.security.ssl.TransportContext.fatal(:-1)
        sun.security.ssl.KeyShareExtension$SHKeyShareConsumer.consume(:-1)
        sun.security.ssl.SSLExtension.consumeOnLoad(:-1)
        sun.security.ssl.SSLExtensions.consumeOnLoad(:-1)
        sun.security.ssl.ServerHello$T13ServerHelloConsumer.consume(:-1)
        sun.security.ssl.ServerHello$ServerHelloConsumer.onServerHello(:-1)
        sun.security.ssl.ServerHello$ServerHelloConsumer.consume(:-1)
        sun.security.ssl.SSLHandshake.consume(:-1)
        sun.security.ssl.HandshakeContext.dispatch(:-1)
        sun.security.ssl.SSLEngineImpl$DelegatedTask$DelegatedAction.run(:-1)
        sun.security.ssl.SSLEngineImpl$DelegatedTask$DelegatedAction.run(:-1)
        java.security.AccessController.doPrivileged(:-1)
        sun.security.ssl.SSLEngineImpl$DelegatedTask.run(:-1)
        java.util.ArrayList.forEach(:-1)
        jdk.internal.net.http.common.SSLFlowDelegate.lambda$executeTasks$3(:-1)
        jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(:-1)
        jdk.internal.net.http.common.SSLFlowDelegate.executeTasks(:-1)
        jdk.internal.net.http.common.SSLFlowDelegate.doHandshake(:-1)
        jdk.internal.net.http.common.SSLFlowDelegate$Reader.processData(:-1)
        jdk.internal.net.http.common.SSLFlowDelegate$Reader$ReaderDownstreamPusher.run(:-1)
        jdk.internal.net.http.common.SequentialScheduler$LockingRestartableTask.run(:-1)
        jdk.internal.net.http.common.SequentialScheduler$CompleteRestartableTask.run(:-1)
        jdk.internal.net.http.common.SequentialScheduler$SchedulableTask.run(:-1)
        java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(:-1)
        java.util.concurrent.ForkJoinTask.doExec(:-1)
        java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(:-1)
        java.util.concurrent.ForkJoinPool.scan(:-1)
        java.util.concurrent.ForkJoinPool.runWorker(:-1)
        java.util.concurrent.ForkJoinWorkerThread.run(:-1)
```

**Comment by niwinz** [comment]
Yep, a known issue, it will be fixed soon with a new docker image

**Comment by ghost** [comment]
perfect, glad to hear :+1:

**Comment by niwinz** [comment]
new images published, pull the latest or the `1.17.0-1` tag

**Comment by ghost** [comment]
Now it works :tada: 
@niwinz Thanks for the fast fix
